include/NovelRT.Interop/Ecs/NrtComponentBufferMemoryContainer.h

```diff
@@ -19,7 +19,7 @@ extern "C"
                                                                                       void* context);
 
     void Nrt_ComponentBufferMemoryContainer_PrepContainerForFrame(NrtComponentBufferMemoryContainerHandle container,
-                                                                  NrtEntityIdVectorHandle entitiesToDelete);
+                                                                  NrtEntityIdVectorHandle entitiesToDestroy);
```

Comment:
Not sure why GitHub deleted my comment on this one - this can be reverted since it reflects a member method in C++.